guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,80 @@
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, restore a virtual machine snapshot.
++
+[IMPORTANT]
+====
+During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+The IP address can change.
+====
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you created snapshosts.
+
+Active and passive {ProjectServer}, with external storage::
+How do I back up?:::
+Store the following critical data on network attached storage: content in `/var/lib/pulp` and database in `/var/lib/pgsql`.
+Replicate this storage into a different data center.
+Attach the storage to a {ProjectServer} that is a clone of the primary {ProjectServer} but runs passively.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, switch DNS records of the active {ProjectServer} with the passive {ProjectServer}.
+This ensures that the passive server becomes the active server.
+All hosts remain connected without configuration updates.
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you synchronized the data storage.
+
+Active and passive {ProjectServer}, with backup and restore::
+How do I back up?:::
+Ensure periodic backups of your {ProjectServer}.
+Copy this backup to a passive {ProjectServer} and restore them on the passive server.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, switch DNS records of the active {ProjectServer} with the passive {ProjectServer}.
+This ensures that the passive server becomes the active server.
+All hosts remain connected without configuration updates.
++
+Lowering your DNS time to live (TTL) value before switching the DNS records will help ensure the DNS records are updated quickly.
+Your TTL affects how long it takes before your hosts are able to reconnect and access the new active {ProjectServer}.
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you took and restored backups and on how long it takes to complete the restore process.
+
+Dual active {ProjectServer}::
+How do I back up?:::
+Operate an active, independent {ProjectServer} per data center.
+Hosts from each data center are registered to the {ProjectServer} in that data center.
+Then configure automation to ensure recovery in case of a disruptive event.
+For example, you can periodically run a health check and if the health check discovers that the current {ProjectServer} a host is registered to does not resolve, the host is re-registered to the other {ProjectServer}.
++
+To minimize downtime, you can automate the recovery in various ways.
+For example, you can use the {Project} Ansible collection.
+For more information, see {AdministeringDocURL}Managing_Project_with_Ansible_Collections_admin[Managing {Project} with Ansible collections] in _{AdministeringDocTitle}_.
```

Comment:
We really should rename this to "Managing Project with Ansible", the "collections" part is a silly implementation detail.

Reply:
https://github.com/theforeman/foreman-documentation/issues/3576